content/general/push/admin.textile

```diff
@@ -29,23 +29,13 @@ The client libraries provide a Push Admin API that is intended to be used on a c
 The push admin API is accessible via the @push@ attribute of the realtime client. For example to publish a push notification directly to a device, you would access the "@publish@":#publish method as follows:
 
 ```[jsall]
-var realtime = new Ably.Realtime({ key: apiKey });
-realtime.push.admin.publish(recipient, data, function(err) {
-  if (err) {
-    console.log('Unable to publish push notification; err = ' + err.message);
-    return;
-  }
-  console.log('Push notification published');
+var rest = new Ably.Rest({ key: apiKey });
+rest.push.admin.publish(recipient, data);
 });
```

Comment:
I think the notification payload is included separately. This example implies that it may be part of the `data` object, whereas it's not. See https://www.ably.io/documentation/rest-api#push-publish

Reply:
Same `notification` payload comment applies to other languages.